CorsixTH/Lua/hospital.lua

```diff
@@ -1388,7 +1388,7 @@ end
 
 !param patient (Patient) The patient that just got treated.
 ]]
-function Hospital:receiveMoneyForTreatment(patient)
+function Hospital:receiveMoneyForTreatment(patient, pay_amount)
   if not self.world.free_build_mode then
     local disease_id = patient:getTreatmentDiseaseId()
     if disease_id == nil then return end
```

Comment:
The second part of Alberth's comment about checking disease later would likely be referring to around here as an `assert`. Asserts resolve bugs faster at the cost of game recovery (`error` can be more recoverable). This doesn't sound like a priority though to look at.